tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -154,6 +154,20 @@ kbs_set_resource_from_file() {
 	kbs-client --url "$(kbs_k8s_svc_http_addr)" config \
 		--auth-private-key "$KBS_PRIVATE_KEY" set-resource \
 		--path "$path" --resource-file "$file"
+
```

Comment:
Interesting so if I understand this correctly the `kbs-client` accepts the request but it might take a while until it resolves it? Wouldn't it be better to fix this on the `kbs-client` side? (eg. introducing `--wait` or such to wait for the resource creation?)

Reply:
Maybe it'd be better fixing on kbs-client code, regardless, till it happens there we need a workaround on our side.

Reply:
ack